diff --git a/Grayjay.Desktop.CEF/Program.cs b/Grayjay.Desktop.CEF/Program.cs
index a741d4f..27f7e2c 100644
--- a/Grayjay.Desktop.CEF/Program.cs
+++ b/Grayjay.Desktop.CEF/Program.cs
@@ -261,7 +261,7 @@ namespace Grayjay.Desktop
             SyncShared.Logger.WillLog = (level) => Logger.WillLog((LogLevel)level);
             SyncShared.Logger.LogCallback = (level, tag, message, ex) => Logger.Log((LogLevel)level, tag, message, ex);
 
-            GrayjayDevSettings.Instance.DeveloperMode = File.Exists("DEV");
+            GrayjayDevSettings.Instance.DeveloperMode = File.Exists(Path.Combine(Directories.Base, "DEV"));
 
             foreach(var arg in args)
                 Console.WriteLine(Logger.FormatLogMessage(LogLevel.Info, nameof(Program), "Arg: " + arg));

