diff --git a/Grayjay.Desktop.CEF/Program.cs b/Grayjay.Desktop.CEF/Program.cs
index a741d4f..aecce73 100644
--- a/Grayjay.Desktop.CEF/Program.cs
+++ b/Grayjay.Desktop.CEF/Program.cs
@@ -358,7 +358,7 @@ namespace Grayjay.Desktop
                 if (OperatingSystem.IsWindows() || OperatingSystem.IsMacOS())
                     cef.Start("--disable-web-security --use-alloy-style --use-native " + userDataDirCmd + extraArgs);
                 else
-                    cef.Start("--disable-web-security --use-alloy-style --use-native --no-sandbox " + userDataDirCmd + extraArgs);
+                    cef.Start("--disable-web-security --use-alloy-style --no-sandbox " + userDataDirCmd + extraArgs);
                 Logger.i(nameof(Program), $"Main: Starting DotCefProcess finished ({startWindowWatch.ElapsedMilliseconds}ms)");
             }
 
