diff --git a/Grayjay.Desktop.CEF/CEFWindowProvider.cs b/Grayjay.Desktop.CEF/CEFWindowProvider.cs
index c6bc368..d4c908d 100644
--- a/Grayjay.Desktop.CEF/CEFWindowProvider.cs
+++ b/Grayjay.Desktop.CEF/CEFWindowProvider.cs
@@ -35,7 +35,7 @@ namespace Grayjay.Desktop.CEF
                 preferredWidth: preferredWidth,
                 preferredHeight: preferredHeight,
                 title: title, 
-                iconPath: Path.GetFullPath("grayjay.png")
+                iconPath: Path.GetFullPath("/app/grayjay/grayjay.png")
             );
 
             await window.SetDevelopmentToolsEnabledAsync(true);
@@ -74,7 +74,7 @@ namespace Grayjay.Desktop.CEF
                 preferredWidth: 385, 
                 preferredHeight: 833, 
                 title: title, 
-                iconPath: Path.GetFullPath("grayjay.png"), 
+                iconPath: Path.GetFullPath("/app/grayjay/grayjay.png"), 
                 developerToolsEnabled: true, 
                 modifyRequests: true,
                 resizable: false,
diff --git a/Grayjay.Desktop.CEF/Program.cs b/Grayjay.Desktop.CEF/Program.cs
index 32fc2b6..7b56aad 100644
--- a/Grayjay.Desktop.CEF/Program.cs
+++ b/Grayjay.Desktop.CEF/Program.cs
@@ -358,7 +358,7 @@ namespace Grayjay.Desktop
                     preferredWidth: 1300,
                     preferredHeight: 950,
                     title: "Grayjay",
-                    iconPath: Path.GetFullPath("grayjay.png"),
+                    iconPath: Path.GetFullPath("/app/grayjay/grayjay.png"),
                     appId: "com.futo.grayjay.desktop"
                 );
                 await window.SetDevelopmentToolsEnabledAsync(true);
